### DISCUSSION TITLE
How do I create items?

### DISCUSSION THREAD

**Discussion by Hugo-Persson** [ghostfolio/ghostfolio#2176] [Q&A]
What is the best way to track value of real estate? Is it possible to create items that I can manually update the current price of. I can't find any docs for it but I saw something about (value) items in another discussion,

**Comment by dtslvr** [comment]
Hi @Hugo-Persson

You can add a real estate by creating an activity with type _Valuable_. The foundation already exists that you can store current market data. But for now a GUI is missing. Please get in touch if you are interested in contributing to this.

**Reply by Hugo-Persson** [reply]
Ok thank you for quick reply @dtslvr . I would love to help develop some GUI for this. I am going on vacation for a week tomorrow so won't have time right now. But I'll contact you when I am back to start work on it, for now I will update DB directly

**Comment by sfuerte** [comment]
out of curiosity, has there been any traction on this one?
